pkg/jobrunaggregator/jobruntablecreator/table_creator.go

```diff
@@ -0,0 +1,37 @@
+package jobruntablecreator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"cloud.google.com/go/bigquery"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+)
+
+type allJobRunTableCreatorOptions struct {
+	ciDataClient jobrunaggregatorlib.CIDataClient
+	ciDataSet    *bigquery.Dataset
+}
+
+func (r *allJobRunTableCreatorOptions) Run(ctx context.Context) error {
+
+	// Create JobRunTable
+	jobRunTable := r.ciDataSet.Table(jobrunaggregatorlib.JobsTableName)
```

Comment:
Are we making the jobRunTable or the jobsTable? Some confusion in this PR referencing jobRuns when I think you're making jobs.

Reply:
hm... you're right.  Let me think about how to change that to be clearer because I'm hoping to also create the `JobRuns` table soon.

Reply:
I renamed things to reflect that we're generating tables (now JobRuns along with Jobs).